recipes-core/packagegroups/packagegroup-ni-runmode.bb

```diff
@@ -40,10 +40,12 @@ RDEPENDS:${PN} = "\
 RDEPENDS:${PN} += "\
 	cpp \
 	cpp-symlinks \
+	firewalld \
```

Comment:
Any reason these changes need to be here and not in [packagegroup-ni-snac.bb](https://github.com/ni/meta-nilrt/blob/nilrt/master/next/recipes-core/packagegroups/packagegroup-ni-snac.bb#L1) (especially since we're not even going to enable them by default)?

Reply:
No; I think you're right. We should try to put this into the runmode package group.

@chaitu236 Our intention in the long term is to switch to nftables and firewalld. I think the NILRT 11.0 release is a reasonable time to introduce those packages, even if we do not enable them until we've had some experience with them in the SNAC configuration.

Reply:
Ok, my concern was that we were increasing BSI size with SNAC-only stuff but if the intention is to have nftables and firewalld in BSI anyway, this is fine.